sunpy/conftest.py

```diff
@@ -114,6 +114,16 @@ def undo_config_dir_patch():
     os.environ["SUNPY_CONFIGDIR"] = oridir
 
 
+@pytest.fixture(scope='session', autouse=True)
+def parfive_header_test(request):
+    """
+    Add a keyword to tell parfive this is a test run.
+    """
+    os.environ["PARFIVE_SUNPY_TESTS"] = "True"
+    yield
+    del os.environ["PARFIVE_SUNPY_TESTS"]
```

Comment:
Was not sure how to do this, so I did this as it was quick.

Reply:
Seems legit to me 👍

Reply:
Yeah this seems good, but why don't we make this a generic environment var for being in our test suite? We might have other non-parfive uses for it in the future?

Reply:
Was not sure what to call it, so I made one up.